cmdopt/src/main/scala/pravda/cmdopt/CommandLine.scala

```diff
@@ -0,0 +1,216 @@
+package pravda.cmdopt
+
+object CommandLine {
+
+  sealed trait Verb[C, +A]
```

Comment:
Can you put all common methods (`text`, `desc`, `read` and etc.) here? They are just the same.

Reply:
I think we can't do this, because they differ in their return value (as we want to chain methods, and a set of methods differs for different instances of Verb).